terraform/modules/function/main.tf

```diff
@@ -145,6 +145,7 @@ module "zip_bucket" {
     "arn:aws:iam::${data.aws_ssm_parameter.prod_account[0].value}:role/delegatedadmin/developer/${var.app}-prod-github-actions",
     "arn:aws:iam::${data.aws_ssm_parameter.sbx_account[0].value}:role/delegatedadmin/developer/${var.app}-sbx-github-actions",
   ] : []
+  
```

Comment:
Sorry to nit pick but could this blank line addition be removed?

Reply:
blank line addition has been removed